stores.html

```diff
@@ -5,8 +5,164 @@
+          </div>
+          <div class="city_item">
+            <label for="City"
+              >City*
+              <br />
+              <select name="City" class="store_select" id="City" value="">
+                <option value="none">Select your city</option>
+                <option value="London">London</option>
+                <option value="Leeds">Leeds</option>
+                <option value="Scotland">Scotland</option>
+                <option value="Liverpool">Liverpool</option>
+                <option value="Manchester">Manchester</option>
+              </select>
+            </label>
+            <label for="postcode">
+              Postcode* <br />
+              <input type="text" class="input" required />
+            </label>
+          </div>
+
+          <div class="color_item">
+            <p>Select a colour</p>
+
+            <label for="form__karmaorange">
+              <input
+                type="radio"
+                id="form__karmaorange"
+                value="orange"
+                name="colour"
+              />
+              Karma orange
+            </label>
+            <label for="form__spacegrey">
+              <input
+                type="radio"
+                id="form__spacegrey"
+                value="grey"
+                name="colour"
+              />
+              Space Grey
+            </label>
+          </div>
+
+          <div class="terms_conditions">
+            <label for="form__terms_conditions">
+              <input
+                type="checkbox"
+                id="form__terms"
+                value="false"
+                name="policy"
+              />
+              By placing your order you agree to Karma's
+              <a href="Terms">Terms and Conditions</a>. *
+            </label>
+          </div>
+
+          <div class="btn_item">
+            <button type="submit" class="store_btn button">
+              Place my order
+            </button>
+          </div>
+        </section>
+        <section class="store_image"></section>
+      </article>
+    </main>
+
+    <footer class="footer">
+      <div class="footer_header">
+        <h3>Join us on</h3>
+      </div>
+
+      <div class="social_icons_container">
+        <div class="social_icons">
+          <a class="social_icons_page" href="https://twitter.com">
+            <img src="./img/twitter-icon.svg" alt="twitter icon" />
+          </a>
+        </div>
+        <div class="social_icons">
+          <a class="social_icons_page" href="https://facebook.com"
+            ><img src="./img/facebook-icon.svg" alt="facebook icon"
+          /></a>
+        </div>
+        <div class="social_icons">
+          <a class="social_icons_page" href="https://instagram.com">
+            <img src="./img/instagram-icon.svg" alt="instagram icon" />
+          </a>
+        </div>
+      </div>
+
+      <div class="copy_right_text">
+        <h4>©Karma Mobility.Inc</h4>
+      </div>
+    </footer>
+  </body>
```

Comment:
* The changes at `<title>` elements are nice 
* Remember the `name` attributes are needed for each submitted form field
* Can further improve on factorising the CSS files